bee-network/src/peer/list.rs

```diff
@@ -337,8 +339,8 @@ impl PeerList {
     }
 
     fn find_peer_if_connected(&self, addr: &Multiaddr) -> Option<PeerId> {
-        #[allow(clippy::filter_next)]
-        self.filter(|info, state| state.is_connected() && info.address == *addr)
+        // #[allow(clippy::filter_next)]
```

Comment:
Does this need to be removed?

Reply:
yes, `clippy` just saw the `filter(...).next()` previously but didn't understand that it's our own impl. of a `filter`. Since I renamed it to `fitler_info`, it doesn't complain anymore, and can be removed.